.travis.yml

```diff
@@ -2,14 +2,28 @@ language: python
 cache: pip
 python:
 - 2.7
-- 3.6
+- 3.5
 
 services:
 - xvfb
 
 env:
 - DJANGO_SETTINGS_MODULE=test_setting  TOXENV=django111-drf37
+- DJANGO_SETTINGS_MODULE=test_setting  TOXENV=django111-drf38
```

Comment:
move 'DJANGO_SETTINGS_MODULE=test_setting' to pytest configs from all those envs